CHANGELOG.md

```diff
@@ -26,6 +26,16 @@ Classify the change according to the following categories:
     ##### Removed
     ### Patches
 
+## Develop - 2025-10-02
```

Comment:
When we merge this, just increment the patch number e.g. 3.15.**1**, since these are really just patch fixes for the /ensite endpoint